packages/api-request/src/middlewares/test/root-url.js

```diff
@@ -0,0 +1,17 @@
+import createRootUrlMiddleware from '../root-url';
+
+describe( 'Root URL middleware', () => {
+	it( 'should append the root URL', () => {
+		const rootURL = 'http://wp.org/wp-admin/rest/';
+		const rootURLMiddleware = createRootUrlMiddleware( rootURL );
+		const requestOptions = {
+			method: 'GET',
+			path: '/wp/v2/posts',
+		};
+		const callback = ( options ) => {
+			expect( options.url ).toBe( 'http://wp.org/wp-admin/rest/wp/v2/posts' );
+		};
+
+		rootURLMiddleware( requestOptions, callback );
```

Comment:
We should add [expect.hasAssertions](https://facebook.github.io/jest/docs/en/expect.html#expecthasassertions) call to make sure that `callback` was executed.